src/assets/image/sidebar/index.ts

```diff
@@ -1,5 +1,8 @@
+export { default as CircleQuestionIcon } from "@assets/image/sidebar/CircleQuestion.svg?react";
```

Comment:
Move the **sidebar** folder to the icons folder.

Reply:
moved to the icons folder